h2o-persist-google-cloud-storage-shadow/build.gradle

```diff
@@ -0,0 +1,26 @@
+description = "Google Cloud Storage library in one single shadow jar, relocated under ai.h2o"
```

Comment:
This will shadow all possible dependencies. Are you sure that is not an issue for any of them?

Reply:
Yup, we were discussing this solution with @michal-raska and it seemed liked easiest and the most manageable solution. I would wait for tests results to tell us more and check if persist-gcs tests passed

Reply:
it also relocates `jodatime` which used to be relocated in h2o (it still might be. If it is there would be a clash of dependencies in the final assembly - not a good thing. Generally speaking, doing this can be dangerous because you do lose control over the dependencies you are using and if you are relocating some of them in your own project you can run into weird issues.

Reply:
Good point. In that case we need to find a way how to relocate just guava. The issue is that a lot of google libraries are using that guava version and we need to to force them to use ours

Reply:
I will change the script to just relocate guava, not everything.

I didn't do this directly as I didn't find a way how to relocate transitive dependencies. But maybe shadowJar has support for that

Reply:
I will think about an alternative solution.

Usually, the relocation should happen at the point where the library causes an issue. This is a bit too early - it might be well justified by our use case but I don't think this is the common pattern.

Reply:
I will try to relocate as part of Sparkling Water